### PR TITLE
Exhibit tweaks 432 433

### LINKED ISSUE #432: Proposal: provide some context when viewing exhibit results?
@caseyedavis12 : Thinking some more about the problem of providing some context when viewing an exhibit result set. What I think might be possible would be to supplement the list of facets at the top.
screen shot 2015-05-22 at 1 24 35 pm

The text there could be the first *x* characters in the description, or we could create a new field just for this.

I'm not super enthusiastic about this, but it probably wouldn't be that hard to do, either.

### LINKED ISSUE #433: Proposal: provide context on individual items in exhibits
screen shot 2015-05-22 at 1 43 37 pm

@caseyedavis12 : This is probably a bad idea, but wanted to put it out there. Instead of just supplying a list of IDs, it could be a hash, with the value of each pair being an in-context description. Just as we add exhibit membership information on ingest, we could add more description to individual records. (Probably record it as another description, with an identifying attribute.) Then it could be shown in the results list, and/or on individual records.

(Individual record pages are state-less: It would be a really bad idea to show it differently, depending on how you got to it.)

### PR DESCRIPTION
**Pull request by mccalluc**
fix #432
fix #433

### CODE REVIEW

**Comment by foo4thought** [comment]
@afred : Chuck and I looked over this and I voted to *not* merge at this time because you ought to see it first.

File: app/models/exhibit.rb [hunk 1]
```diff
@@ -29,18 +34,28 @@ def pop(hash, key)
...
   
+  @@exhibits_by_name = Hash[
+    YAML.load_file(Rails.root + 'config/exhibits.yml').map do |hash|
+      [hash['name'], Exhibit.new(hash)]
+    end
+  ]
+  
   @@exhibits_by_item_id = Hash[
     Exhibit.all.map{ |exhibit| exhibit.ids }.flatten.uniq.map do |id|
       [id, Exhibit.all.select { |exhibit| exhibit.ids.include?(id) } ]
```
Comment (afred) [inline app/models/exhibit.rb]: @mccalluc does `exhibit.ids` still exist? It looks like you removed it in the YAML file below.

Comment (mccalluc) [inline app/models/exhibit.rb]: So, in order to let you cram a little description on each one, I changed it from a list of IDs to a hash. The `ids` method still works: it is just getting the keys.

(I was on the fence about even the idea of this per-item adhoc metadata; I wouldn't fight hard for it if you don't think it's the right direction.)

Comment (afred) [inline app/models/exhibit.rb]: ah.. got it. I'm ok with this solution. I prefer it to sticking exhibit-specific-metadata in the AMS.

I just missed line #'s [8](https://github.com/WGBH/AAPB2/blob/exhibit-tweaks-432-433/app/models/exhibit.rb#L8) and [38](https://github.com/WGBH/AAPB2/blob/exhibit-tweaks-432-433/app/models/exhibit.rb#L38).

**Pull request merged**